extensions/wikia/VideoHandlers/apiwrappers/ApiWrapper.class.php

```diff
@@ -196,32 +191,7 @@ protected function checkForResponseErrors( $status, $content, $apiUrl ){
 	}
 
 	protected function processResponse( $response ){
```

Comment:
We could remove this method and just leave `postProcess`, what do you think?

Reply:
Why? It does process the response :)